### ISSUE TITLE
Connection host must be valid Host without http

### ISSUE THREAD

**Issue by pixxon** [Radarr/Radarr#6124]
**Describe the bug**

Most of the connections ( or at least the ones I use/checked ) only allow host to be specified. This limits the options especially when the services are running behind a reverse proxy under different baseurls.

**To Reproduce**
1. Go to 'Settings -> Connect'
2. Click on 'New Connection'
3. Select 'Emby'
4. Set Name to anything and Host to '192.168.0.129/emby'
5. Click 'Test'
4. See error --> "must be valid Host without http://"

**Expected behavior**

Services running under baseurl should be connectable by Radarr.

**Screenshots**

![image](https://user-images.githubusercontent.com/12481929/112829678-c08c2600-9091-11eb-997f-7116a4307d3f.png)

**Platform Information (please complete the following information):**
 - OS: Ubuntu 20.04 (3.0.0.4136-ls2 by linuxserver.io)
 - Docker: Yes
 - Mono or .NET Version (System -> Status): .NET Core (3.1.10)
 - Browser and Version (Only needed for UI issues): N/A
 - Radarr Version: 3.0.0.4136
 - Radarr Branch: master

**Comment by pixxon** [comment]
Either allow host to be a proper uri here: https://github.com/Radarr/Radarr/blob/develop/src/NzbDrone.Core/Notifications/MediaBrowser/MediaBrowserSettings.cs#L13

Or add base url like in case of https://github.com/Radarr/Radarr/issues/3109

**Comment by ta264** [comment]
Enable advanced settings

**Comment by ta264** [comment]
Didn't read enough, sorry

**Comment by pixxon** [comment]
> Enable advanced settings

If the solution to introduce a baseurl under advanced settings is fine, I could implement it for emby/jellyfin.

**Comment by ta264** [comment]
I didn't think emby supported a urlbase, can't see it in settings on my instance. Are you trying to run it through a reverse proxy?

**Comment by pixxon** [comment]
Yes, I am using Traefik as my reverse proxy to access emby under [IP]/emby.
There is no need to set baseurl for emby, because it can detect it automatically and work out of the box.

**Comment by ta264** [comment]
I suggest not running these connections through traefik, it adds an unnecessary point of failure. Just point radarr straight at emby. Join the discord if you need help with this - this is a niche request so doubt it will be fixed in the immediate future.

**Comment by pixxon** [comment]
Since I can address emby directly with the port 8096, I can do a workaround this.
However, I plan to switch Jellyfin, where currently I have to specify the base url. I also like to avoid publishing services via ports, thats why I'm using Traefik in the first place.

As I said above, if acceptable I can introduce baseurl for emby/jellyfin in the scope of this ticket. Just wanted somebody from the Radarr team to agree with the solution.
